docs/topics/Community_Resources.md

```diff
@@ -47,6 +47,7 @@ Many of these libraries are represented in the [unofficial, community-driven Dis
 - [discord-interactions-python](https://github.com/discord/discord-interactions-python)
 - [discord-interactions-php](https://github.com/discord/discord-interactions-php)
 - [discord-slash-commands](https://github.com/MeguminSama/discord-slash-commands)
+- [slash-command-generator] (https://github.com/iSaluki/discord-slash-commands)
```

Comment:
remove the space
```suggestion
- [slash-command-generator](https://github.com/iSaluki/discord-slash-commands)
```

Reply:
Looks Great! 👍  
Looking forward to using it.

Reply:
> Looks Great! 👍  
> Looking forward to using it.

Thanks! It's live now on my GitHub. Just let me know if you have any problems or feature requests.